StepDefinitions/LanguageStep.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyModel;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using SpecFlowProject_QA_MARS.Pages;
+using SpecFlowProject_QA_MARS.Utils;
+
+namespace SpecFlowProject_QA_MARS.StepDefinitions
+{
+
+    [Binding]
+    public class LanguageLevel : Start
+    {
+        Languages LangObj = new Languages();
```

Comment:
Please make use of constructors https://www.knowledgehut.com/tutorials/csharp/csharp-constructors